lib/dl_connector_clickhouse/dl_connector_clickhouse/formula/definitions/functions_string.py

```diff
@@ -130,6 +130,12 @@
             ),
         ]
     ),
+    # regexp_extract_all
+    base.FuncRegexpExtractAll(
+        variants=[
+            V(D.CLICKHOUSE, lambda text, pattern: sa.func.extractAll(sa.func.assumeNotNull(text), pattern)),
```

Comment:
https://clickhouse.com/docs/en/sql-reference/functions/functions-for-nulls#assumenotnull
kinda scared of "Returned values: An arbitrary value, if the input value is NULL". can we wrap the case of `text` being `NULL`?
also, definitely need a test for that

Reply:
this functions wraps null text to ""

Reply:
added test